features/customers.feature

```diff
@@ -133,6 +133,28 @@ Scenario: List all customers
     And I should see "user3" in the customer results
     And I should see "user4" in the customer results
 
+Scenario: Update a customer
+    When I visit the "Home Page"
+    And I set the "First Name" to "Tim"
+    And I set the "Last Name" to "Giovani"
+    And I press the "Search for Customer" button
+    Then I should see the message "Success"
+    And I should see "Tim" in the "First Name" field
+    And I should see "Giovani" in the "Last Name" field
+
+    # Update this cusotmer
+    When I press the "Retrieve" button
```

Comment:
I think we can just remove this Retrieve line here, since I think the customer is already retrieved from the previous steps